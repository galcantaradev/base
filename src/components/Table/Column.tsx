import { ReactNode } from 'react';

export type ColumnProps<T> = {
  header?: string;
  data: (value: T) => ReactNode;
};

export const Column = <T extends any>(props: ColumnProps<T>) => {
  return null;
};
