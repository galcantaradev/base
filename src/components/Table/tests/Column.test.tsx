import { render } from '@testing-library/react';
import React from 'react';

import { Column } from '../Column';

describe('Column.test.tsx', () => {
  it('matches the snapshot', () => {
    const { asFragment } = render(<Column data={jest.fn()} />);

    expect(asFragment()).toMatchSnapshot();
  });
});
