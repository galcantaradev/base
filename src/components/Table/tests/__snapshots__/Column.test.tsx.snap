// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Column.test.tsx matches the snapshot 1`] = `<DocumentFragment />`;
